google/cloud/spanner/internal/database_admin_retry.cc

```diff
@@ -54,11 +56,42 @@ std::unique_ptr<BackoffPolicy> DefaultAdminBackoffPolicy() {
       .clone();
 }
 
+#ifndef GOOGLE_CLOUD_CPP_SPANNER_ADMIN_DEFAULT_POLLING_TIMEOUT
```

Comment:
These are settings that we need to tweak at build time? How often do these change?

Reply:
> These are settings that we need to tweak at build time?

Our users might.

> How often do these change?

Approximately never.

I do not like libraries that hard-code things for me, but I am happy to remove this if you think it is too much.

Reply:
No change necessary; keeping these is totally fine with me.

But I'll clarify my thinking on this topic a little.

These are only user-configurable if the user is compiling the source from scratch. If they are linking against a library, these defines will be ignored and the user will get whatever settings the library maintainer compiled the library with. 

Since these are just the settings for the default policy, users can already customize the timeouts by using a non-default policy.

I'd be happy to omit these preprocessor knobs until someone needs them. But I'm also totally happy to keep them if you see them being useful.

(I imagine you may now feel pressure to remove them to appease the reviewer. Please don't :-) Really, feel free to keep them unless you agree. Either is fine with me. My approval stands as-is.)